### PR TITLE
fix unused warnings

### PR DESCRIPTION
**Pull request by SethTisue**
there isn't a 2.12.2 nightly quite yet with scala/scala#5402 in it,
but I tested with ++2.12.2-bin-bad61ce-SNAPSHOT

~~the sbt version bump is unrelated~~ (removed)

### CODE REVIEW

**Comment by SethTisue** [comment]
well this is alarming, @dwijnand does this alarm you?

```
[info] Set current project to scala-partest (in build file:/home/travis/build/scala/scala-partest/)
Could not create file /home/travis/build/scala/scala-partest/target/streams/$global/projectDescriptors/$global/streams/outjava.io.IOException: No such file or directory
	at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
	at sbt.IO$.touch(IO.scala:210)
	at sbt.std.Streams$$anon$3$$anon$2.make(Streams.scala:129)
	at sbt.std.Streams$$anon$3$$anon$2.text(Streams.scala:113)
	at sbt.std.Streams$$anon$3$$anon$2.log(Streams.scala:124)
	at sbt.std.TaskStreams$class.log(Streams.scala:56)
	at sbt.std.Streams$$anon$3$$anon$2.log$lzycompute(Streams.scala:102)
	at sbt.std.Streams$$anon$3$$anon$2.log(Streams.scala:102)
	at sbt.Classpaths$$anonfun$depMap$1.apply(Defaults.scala:1613)
	at sbt.Classpaths$$anonfun$depMap$1.apply(Defaults.scala:1612)
	at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:35)
	at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:34)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: No such file or directory
	at java.io.UnixFileSystem.createFileExclusively(Native Method)
	at java.io.File.createNewFile(File.java:1012)
	at sbt.IO$$anonfun$1.apply$mcZ$sp(IO.scala:210)
	at sbt.IO$$anonfun$1.apply(IO.scala:210)
	at sbt.IO$$anonfun$1.apply(IO.scala:210)
	at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
	at sbt.IO$.touch(IO.scala:210)
	at sbt.std.Streams$$anon$3$$anon$2.make(Streams.scala:129)
	at sbt.std.Streams$$anon$3$$anon$2.text(Streams.scala:113)
	at sbt.std.Streams$$anon$3$$anon$2.log(Streams.scala:124)
	at sbt.std.TaskStreams$class.log(Streams.scala:56)
	at sbt.std.Streams$$anon$3$$anon$2.log$lzycompute(Streams.scala:102)
	at sbt.std.Streams$$anon$3$$anon$2.log(Streams.scala:102)
	at sbt.Classpaths$$anonfun$depMap$1.apply(Defaults.scala:1613)
	at sbt.Classpaths$$anonfun$depMap$1.apply(Defaults.scala:1612)
	at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:35)
	at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:34)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] (*:projectDescriptors) Could not create file /home/travis/build/scala/scala-partest/target/streams/$global/projectDescriptors/$global/streams/outjava.io.IOException: No such file or directory
```

**Comment by SethTisue** [comment]
trying again now without the sbt version bump

**Comment by sjrd** [comment]
That happens to me occasionally. `clean` or just retry fixes it. I think it's a race on the file system with other stuff such as an IDE.

**Comment by eed3si9n** [comment]
Tracked in https://github.com/sbt/sbt/issues/2863.

**Comment by SethTisue** [comment]
new PR with just the sbt bump: https://github.com/scala/scala-partest/pull/84

File: src/main/scala/scala/tools/partest/nest/FileManager.scala [hunk 1]
```diff
@@ -8,16 +8,7 @@
 package scala.tools.partest
 package nest
 
-import java.io.{
-  File,
-  IOException,
```
Comment (som-snytt) [inline src/main/scala/scala/tools/partest/nest/FileManager.scala:13]: Missed opportunity for trailing mccomma.

Comment (SethTisue) [reply src/main/scala/scala/tools/partest/nest/FileManager.scala:13]: don't count your trailing comma chickens before the 2.12.2 egg has hatched!

**Pull request merged**